t/pgtap/first_release_dates.sql

```diff
@@ -225,6 +225,149 @@ SELECT results_eq(
   'VALUES (1970::smallint, 11::smallint, null::smallint)'
 );
 
+UPDATE release SET status = 6 WHERE id = 3;
+
+SELECT results_eq(
+  'all_release_first_release_dates',
+  $$
+    VALUES
+      (1::integer, 1999::smallint, 1::smallint, 4::smallint),
+      (2::integer, 1989::smallint, null::smallint, null::smallint)
+  $$,
+  'Release dates as expected after marking release as cancelled'
```

Comment:
It’s not about the “release dates”, it’s about the first release date for releases of the same group.
```suggestion
  'Release’s first release (of the same group) dates as expected after marking release as cancelled'
```

Reply:
IIRC this is actually about the earliest date for *this specific release* (for cases when there's several regions with different dates), or am I remembering wrong? (so it's not about a group).

Reply:
Good catch. Then maybe something like that instead?
```suggestion
  'Release’s first release event dates as expected after marking release as cancelled'
```